workers/unity/Packages/io.improbable.gdk.buildsystem/Util/CommandlineParser.cs

```diff
@@ -43,13 +44,13 @@ public static List<BuildTarget> GetBuildTargetFilter(CommandLineArgs args)
                         case "macos":
                             return BuildTarget.StandaloneOSX;
                         default:
-                            throw new BuildFailedException(
-                                "Unknown build target value: " + buildTargetFilterArg);
+                            throw new ArgumentOutOfRangeException(nameof(target), target,
```

Comment:
There's more of these changes (`BuildFailedException` --> `ArgumentOutOfRangeException`) in this file 😉